app/views/posts/_expanded.html.erb

```diff
@@ -100,7 +100,10 @@
 
       <div class="post--body">
         <%= raw(sanitize(post.body, scrubber: scrubber)) %>
-  
+
+        <% been_edited = post.last_activity_by_id != nil %>
+        <% if (been_edited) then last_edited_by_self = post.user_id == post.last_activity_by_id end %>
```

Comment:
Not blocking on this, but you don't need the parens around the `if` condition - Ruby doesn't require them. It won't break, because it just makes it a subclause of the condition (i.e. same effect), but it's not necessary.